code/modules/instruments/stationary.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "Something broke, contact coderbus."
 	interaction_flags_atom = INTERACT_ATOM_ATTACK_HAND | INTERACT_ATOM_UI_INTERACT | INTERACT_ATOM_REQUIRES_DEXTERITY
 	var/can_play_unanchored = FALSE
-	var/list/allowed_instrument_ids
+	var/list/allowed_instrument_ids = list("r3grand","r3harpsi","crharpsi","crgrand1","crbright1", "crichugan", "crihamgan","piano")
```

Comment:
Turn this into a string list during init.
Look in the code for examples of usage of the `string_list` proc.